### PR TITLE
Feature/muwm 4237

### PR DESCRIPTION
**Pull request by kbui98**
https://jira.cac.washington.edu/browse/MUWM-4237

### CODE REVIEW

File: myuw/data/category_links_import.csv [hunk 1]
```diff
@@ -1,39 +1,41 @@
-Category,Subcategory,affiliation,Central (not campus specific),Central Title,Seattle,Seattle (link text),Bothell (destination URL),Bothell (link text),Tacoma,Tacoma Title,Open in a new tab?,,,,,,,,,,,,,
-PageAcademics,Online Learning,all,,,https://academicsupport.uw.edu/online-learning/,Online Learning Tips and Resources,,,,,no,,,,,,,,,,,,,
```
Comment (fanglinfang) [inline myuw/data/category_links_import.csv:2]: What happens to these empty fields?

File: myuw/data/category_links_import.csv [hunk 2]
```diff
@@ -1,39 +1,41 @@
...
-PageAcademics,Tools & Software,all,http://www.linkedinlearning.uw.edu/,LinkedIn Learning,,,,,,,yes,,,,,,,,,,,,,
-PageCalendar,Other Calendars,all,,,http://www.washington.edu/students/reg/calendar.html,All Academic Calendars,http://www.uwb.edu/calendar/academic,Bothell Academic Calendar,https://www.tacoma.uw.edu/uwt/registrar/academic-calendar,Tacoma Academic Calendars,no,,,,,,,,,,,,,
-PageCalendar,Other Calendars,all,,,https://www.washington.edu/calendar/,Campus Event Calendar,https://www.uwb.edu/calendar,Bothell Campus Event Calendar,https://www.tacoma.uw.edu/home/events-calendar,Tacoma Campus Event Calendar,no,,,,,,,,,,,,,
-PageCalendar,Other Calendars,all,,,https://hub.washington.edu/hub-events/event-calendars/,HUB Event Calendar,https://www.uwb.edu/sea/events,Student Engagement & Activities Events,,,yes,,,,,,,,,,,,,
-PageCalendar,Other Calendars,all,https://isc.uw.edu/workday-payroll-calendar/,Payroll Calendar,,,,,,,yes,,,,,,,,,,,,,
+Category,Subcategory,affiliation,Central (not campus specific),Central Title,Seattle,Seattle (link text),Bothell (destination URL),Bothell (link text),Tacoma,Tacoma Title,Open in a new tab?
```
Comment (fanglinfang) [inline myuw/data/category_links_import.csv:1]: What are the changes we intend to make in this file?

Comment (kbui98) [reply myuw/data/category_links_import.csv:1]: Adding links to GSuIte and Office 365 apps to the Tools & Software section of the Academics Page

Comment (fanglinfang) [reply myuw/data/category_links_import.csv:1]: Then changes should be only those new links.

Comment (kbui98) [reply myuw/data/category_links_import.csv]: I think I fixed it? Can you verify :D

**Review by fanglinfang — APPROVED** [review]

**Review by wmwash — APPROVED** [review]

**Pull request merged**